### PR TITLE
doc updates for new repo layout

### PR DESCRIPTION
**Pull request by dereklwood**
updates to repo instructions to reflect the new repo layouts

### CODE REVIEW

File: coopr-docs/docs/source/installation/packages.rst [hunk 1]
```diff
@@ -162,20 +162,20 @@ RPM using Yum
 -------------
 Download the Cask Yum repo definition file::
 
-  sudo curl -o /etc/yum.repos.d/cask.repo http://repository.cask.co/downloads/centos/6/x86_64/cask.repo
+  sudo curl -o /etc/yum.repos.d/coopr.repo http://repository.cask.co/centos/6/x86_64/coopr/0.9/coopr.repo
```
Comment (awholegunch) [inline coopr-docs/docs/source/installation/packages.rst]: Do you want 0.9 hardcoded, or is this a version that should be a replace and change with each release?

Comment (wolf31o2) [inline coopr-docs/docs/source/installation/packages.rst]: It's fine being hard-coded

**Comment by awholegunch** [comment]
LGTM. I'll let Chris/David sign off on content.

**Comment by wolf31o2** [comment]
Content looks good. I am going to take this over from Derek, so I'll let someone else approve.

**Pull request merged**